tests/ttnn/integration_tests/yolov4/test_ttnn_downsample1.py

```diff
@@ -0,0 +1,219 @@
+            )
+
+            ttnn_module_args.c6["math_fidelity"] = ttnn.MathFidelity.LoFi
+            ttnn_module_args.c6["use_shallow_conv_variant"] = (
+                False if device.arch() == tt_lib.device.Arch.WORMHOLE_B0 else True
+            )
+            ttnn_module_args.c6["dtype"] = ttnn.bfloat8_b
+            ttnn_module_args.c6["weights_dtype"] = ttnn.bfloat8_b
+            ttnn_module_args.c6["activation"] = "relu"  # Fuse relu with conv1
+            ttnn_module_args.c6["deallocate_activation"] = True
+            ttnn_module_args.c6["conv_blocking_and_parallelization_config_override"] = None
+
+            conv6_weight, conv6_bias = fold_batch_norm2d_into_conv2d(model.c6, model.b6)
+            update_ttnn_module_args(ttnn_module_args.c6)
+            parameters["c6"], c6_parallel_config = preprocess_conv2d(
+                conv6_weight, conv6_bias, ttnn_module_args.c6, return_parallel_config=True
+            )
+
+            ttnn_module_args.c7["math_fidelity"] = ttnn.MathFidelity.LoFi
+            ttnn_module_args.c7["use_shallow_conv_variant"] = (
+                False if device.arch() == tt_lib.device.Arch.WORMHOLE_B0 else True
+            )
+            ttnn_module_args.c7["dtype"] = ttnn.bfloat8_b
+            ttnn_module_args.c7["weights_dtype"] = ttnn.bfloat8_b
+            ttnn_module_args.c7["activation"] = "relu"  # Fuse relu with conv1
+            ttnn_module_args.c7["deallocate_activation"] = True
+            ttnn_module_args.c7["conv_blocking_and_parallelization_config_override"] = None
+
+            conv7_weight, conv7_bias = fold_batch_norm2d_into_conv2d(model.c7, model.b7)
+            update_ttnn_module_args(ttnn_module_args.c7)
+            parameters["c7"], c7_parallel_config = preprocess_conv2d(
+                conv7_weight, conv7_bias, ttnn_module_args.c7, return_parallel_config=True
+            )
+
+            ttnn_module_args.c8["math_fidelity"] = ttnn.MathFidelity.LoFi
+            ttnn_module_args.c8["use_shallow_conv_variant"] = (
+                False if device.arch() == tt_lib.device.Arch.WORMHOLE_B0 else True
+            )
+            ttnn_module_args.c8["dtype"] = ttnn.bfloat8_b
+            ttnn_module_args.c8["weights_dtype"] = ttnn.bfloat8_b
+            ttnn_module_args.c8["activation"] = "relu"  # Fuse relu with conv1
+            ttnn_module_args.c8["deallocate_activation"] = True
+            ttnn_module_args.c8["conv_blocking_and_parallelization_config_override"] = None
+
+            conv8_weight, conv8_bias = fold_batch_norm2d_into_conv2d(model.c8, model.b8)
+            update_ttnn_module_args(ttnn_module_args.c8)
+            parameters["c8"], c8_parallel_config = preprocess_conv2d(
+                conv8_weight, conv8_bias, ttnn_module_args.c8, return_parallel_config=True
+            )
+
+        return parameters
+
+    return custom_preprocessor
+
+
+@skip_for_wormhole_b0()
+def test_downsample1(reset_seeds, device):
```

Comment:
`def test_downsample1(device, reset_seeds):`
 and remove `device_id = 0`

Reply:
This is still not removed @ankitmcw please update the commit. Changes are not reflected I suppose.